emission/storage/decorations/token_queries.py

```diff
@@ -11,18 +11,3 @@ def get_all_tokens():
 
 def insert(entry):
     edb.get_token_db().insert_one(entry)
-
-def insert_from_json(jsonfile, userid=0):
```

Comment:
nit: I would probably define a `bulk_insert` here as well and call that from the script, since the bulk insert is the common case.